change/@ni-nimble-components-fd0f6918-b425-44fe-b644-17e155870de3.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
@fredvisser: 
FYI - here is a PR with the `js-beautify` quick fix. If you have an alternative approach, feel free to kill this one.

Reply:
Yea lets ship it if it works. If we find issues can work on re-integrating prettier.